src/ApiTestCase.php

```diff
@@ -107,7 +107,10 @@ public function setUpDatabase()
 
     public function tearDown()
     {
-        if (null !== $this->client && null !== $this->client->getContainer()) {
+        if (null !== $this->client &&
+            null !== $this->client->getContainer() &&
+            $this->client->getContainer() instanceof MockContainerInterface
```

Comment:
```suggestion
            $this->client->getContainer() instanceof \PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer
```

This way we don't need the additional interface.